app/views/homepage/index.html.erb

```diff
@@ -70,9 +70,9 @@
               </p>
               <p>
                 Here you can see all <a href="/news-and-communications">news and communications</a>,
-                <a href="/government/publications">publications</a>,
                 <a href="/government/statistics">statistics</a>
-                and <a href="/government/publications?publication_filter_option=consultations">consultations</a>.
+                and
+                <a href= "<%= URI::encode '/search/policy-papers-and-consultations?content_store_document_type[]=open_consultations&content_store_document_type[]=closed_consultations'%>">consultations</a>.
```

Comment:
`URI::encode` is an alias for the `escape` method, which is deprecated. Perhaps `CGI.escape` instead?